site/_guides/external-authorization.md

```diff
@@ -3,8 +3,6 @@ title: External Authorization Support
 layout: page
 ---
 
-<div id="toc" class="navigation"></div>
```

Comment:
Why remove this?

I can see that it doesn't display correctly on the published site, but it did work locally. This doc is long enough that it needs this index.

Reply:
There's no index, just a box that gets rendered that's empty: 
![image](https://user-images.githubusercontent.com/1048184/95260015-090b0b00-07f6-11eb-963a-3ae232344a89.png)

Reply:
![Screen Shot 2020-10-07 at 8 11 01 am](https://user-images.githubusercontent.com/9917/95260572-af95f100-0874-11eb-93c1-d59640e9e62f.png)

Locally it looks like this. There's some sort of javascript problem on the published site.

Reply:
We can remove the TOC here and file a follow-up issue.